app/views/users/show.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1>Users#show</h1>
-<p>Find me in app/views/users/show.html.erb</p>
+<% provide(:title, @user.name) %>
+<div class="container pt-4">
+    <div class="row">
+        <div class="col-3">
+            <section>
+                <h1>
+                <%= @user.name %><br>
+                <%= link_to user_avatar(current_user, size: 35), current_user %>
```

Comment:
Use @user not current_user. Remove the size:35 and target a class like
  <%= link_to user_avatar(@user, "user-profile"), current_user %>

and you just need to target the user-profile to your css